src/Console/Output/Informer.php

```diff
@@ -32,11 +32,12 @@ public function __construct(OutputWatcher $output)
 
     public function startTask(Task $task)
     {
+        if ($task->isShallow()) {
+            return;
+        }
+
         $this->startTime = round(microtime(true) * 1000);
```

Comment:
But we loose startTime here.

Reply:
we use startTime only in one place inside this file and this case has the same IF guard

Reply:
How to proceed?